baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/OptionalOrElseMethodInvocation.java

```diff
@@ -46,8 +45,13 @@
             .onExactClass("java.util.Optional")
             .named("orElse");
 
-    private static final Matcher<Tree> METHOD_INVOCATIONS =
-            Matchers.contains(ExpressionTree.class, MethodMatchers.anyMethod());
+    private static final Matcher<ExpressionTree> METHOD_OR_CONSTRUCTOR = Matchers.anyOf(
+            MethodMatchers.anyMethod(),
+            MethodMatchers.constructor());
+
+    private static final Matcher<ExpressionTree> METHOD_INVOCATIONS = Matchers.anyOf(
+            METHOD_OR_CONSTRUCTOR,
```

Comment:
This is required because while the symbol for a method invocation is a `MethodSymbol`, the symbol for a constructor invocation is just a `TypeSymbol`. Thus `Matchers.contains(...)` won't detect a expression that is just a constructor invocation.